browser/components/MarkdownPreview.js

```diff
@@ -128,7 +128,7 @@ h1, h2 {
 }
 
 h1 {
-  padding-bottom: 4px;
+  padding-bottom: 0.2em;
```

Comment:
Can you do the same thing for `padding-top`? Because if you change the zoom level to 80% the bottom line will disappear.

Reply:
Just made change, thanks!

Reply:
Hmm...Why you add `padding-top` to `p` instead of `h1`? Can you explain for me please?